src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20160428_entity_attrs_autoinc.xml

```diff
@@ -0,0 +1,372 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <!--Remove some foreign keys-->
+
+    <changeSet logicalFilePath="dummy" author="mbemis" id="entity-attrs-autoinc-1">
+        <sql stripComments="true">
+            alter table ATTRIBUTE drop foreign key ATTRIBUTE_ibfk_1;
```

Comment:
I am concerned that these auto generated foreign key names might be different on prod, double check them.

Name the foreign keys you create below to avoid this in the future.